mozilla_django_oidc/auth.py

```diff
@@ -77,22 +77,25 @@ def verify_claims(self, claims):
 
     def create_user(self, claims):
         """Return object for a newly created user account."""
-        # bluntly stolen from django-browserid
-        # https://github.com/mozilla/django-browserid/blob/master/django_browserid/auth.py
 
-        username_algo = import_from_settings('OIDC_USERNAME_ALGO', None)
         email = claims.get('email')
         if not email:
             return None
 
+        username = self.get_username(claims)
+
+        return self.UserModel.objects.create_user(username, email)
+
+    def get_username(self, claims):
+        """Get the local username based on claims."""
```

Comment:
I am not sure if ``local username`` makes a lot of sense. I would prefer something like:

```Generate username based on claims.```